README.md

```diff
@@ -67,4 +67,71 @@ console.log(todos.todosDisplay);
   'publish to npm - done',
   'write a blog post about carmi - not done' ]
 */
-```
\ No newline at end of file
+```
+
+## Usage with Babel
+
+Compiles Carmi files (`xxx.carmi.js`) automatically using Babel. It reserves the external modules, required by `require`,
```

Comment:
=> it doesn't support `import` right now, but as far as I know, CARMI doesn't as well.